style.css

```diff
@@ -0,0 +1,901 @@
+  }
+
+  .second-section .container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 90%;
+  }
+
+  .second-section .card {
+    flex-direction: column;
+    height: 26vh;
+    justify-content: space-evenly;
+    align-items: center;
+    padding: 1rem;
+  }
+
+  .third-section .card {
+    width: 40%;
+    height: 20vw;
+    font-size: small;
+    padding: 1rem;
+  }
+
+  .second-section .card:hover {
+    border: 1px solid white;
+    opacity: 1;
+  }
+
+  .second-section h2 {
+    width: 100%;
+    margin: 1rem;
+  }
+
+  .hr-about-1 {
+    width: 9%;
+    margin-top: -2rem;
+  }
+
+  .card .content {
+    width: 100%;
+    font-size: small;
+    font-weight: 400;
+    text-align: center;
+    padding: 0.3rem;
+  }
+
+  .card img {
+    width: 50%;
+  }
+
+  .card .title {
+    font-size: medium;
+    font-weight: 400;
+    width: 100%;
+  }
+
+  a,
+  a:hover,
+  a:focus,
+  a:active,
+  a:visited {
+    text-decoration: none;
+  }
+
+  .second-section a {
+    display: block;
+    color: white;
+    text-decoration: underline;
+    text-transform: uppercase;
+    margin: 2rem;
+    transition: all 0.7s 0s ease;
+  }
+
+  .second-section a:hover {
+    color: #ec5342;
+  }
+
+  .second-section button {
+    display: none;
+  }
+
+  /* =========  Thired Section  ================= */
+  .third-section {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .third-section h2 {
+    width: 100%;
+  }
+
+  .third-section .card img {
+    width: 40%;
+  }
+
+  .more-btn {
+    display: none;
+  }
+
+  .featured {
+    width: 80%;
+    text-align: center;
+    margin: 0 auto;
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .featured .hr-3 {
+    width: 5%;
+    color: #ec5342;
+  }
+
+  /* ============ About =================== */
+  .about-section {
+    background-image: url(img/about_bg_01.png);
+    background-size: contain;
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 60vh;
+  }
+
+  .about-section .hr-about-1 {
+    margin: 0 auto;
+    width: 3%;
+  }
+
+  .h1-global {
+    display: none;
+  }
+
+  .h2 {
+    font-size: 40px;
+    font-weight: 900;
+    width: 65%;
+    margin-top: 4rem;
+  }
+
+  .article-1-about {
+    margin-top: 2rem;
+    width: 80%;
+    height: auto;
+  }
+
+  .date-about {
+    font-size: medium;
+  }
+
+  .hr-about-2 {
+    color: #ec5342;
+    width: 3%;
+    margin: 0 auto;
+  }
+
+  .hr-about-4 {
+    color: #ec5342;
+    width: 3%;
+    margin: 0 auto;
+    margin-bottom: 0.5rem;
+  }
+
+  .hr-about-15 {
+    width: 3%;
+    margin: 0 auto;
+    color: #ec5342;
+  }
+
+  .art-logo {
+    width: auto;
+  }
+
+  .articl-2 {
+    margin-bottom: 2rem;
+  }
+
+  .footer-h {
+    display: none;
+  }
+
+  .hr-about {
+    display: none;
+  }
+
+  .img-big-container {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
```

Comment:
- [ ] Please make sure the following cards on the about page are equal in size and properly aligned.
![image](https://user-images.githubusercontent.com/92215878/192033904-a6202381-b0f8-4b9d-a091-5356e8c88175.png)